core/cluster.cxx

```diff
@@ -766,9 +777,9 @@ class cluster_impl : public std::enable_shared_from_this<cluster_impl>
           });
         }
       }
-      session_ = io::mcbp_session(id_, ctx_, tls_, origin_, dns_srv_tracker_);
+      session_ = io::mcbp_session(id_, {}, ctx_, tls_, origin_, dns_srv_tracker_);
     } else {
-      session_ = io::mcbp_session(id_, ctx_, origin_, dns_srv_tracker_);
+      session_ = io::mcbp_session(id_, {}, ctx_, origin_, dns_srv_tracker_);
     }
     session_->bootstrap([self = shared_from_this(), handler = std::move(handler)](
```

Comment:
I think we want to allow the `app_telemetry_reporter` to get update from the cluster session as well.  Also, calling the reporter's `update_config()` method w/ the config after bootstrap allows for cluster only operations to collect telemetry.

I cannot see how I can add my comment on the line I want, but it would be after 808.

Reply:
Fixed